src/components/card.css

```diff
@@ -0,0 +1,4 @@
+.card-default {
+    padding: 1rem;
+    background-color: white;
+}
```

Comment:
Why is the need to make card-default class? when we have card property added in index.css.
Also what is the need for card.css. We can include all generic style properties in index.css

Reply:
it was mentioned to have these two styles in issue ,ok i will include its in index.css